src/govuk/common.mjs

```diff
@@ -34,18 +40,24 @@ export function generateUniqueID () {
 }
 
 /**
- * Config flattening function. Takes any number of objects, flattens them into
- * namespaced key-value pairs, (e.g. {'i18n.showSection': 'Show section'}) and
- * combines them together, with greatest priority on the LAST item passed in.
+ * Config flattening function
+ *
+ * Takes any number of objects, flattens them into namespaced key-value pairs,
+ * (e.g. {'i18n.showSection': 'Show section'}) and combines them together, with
+ * greatest priority on the LAST item passed in.
  *
- * @param {...Object} - Any number of objects to merge together.
- * @returns {Object} - A flattened object of key-value pairs.
+ * @returns {object} A flattened object of key-value pairs.
```

Comment:
Just want to check my understanding of the removal of `@param` – presumably ESLint doesn't like the fact that we had `@param` annotations for params that weren't in the actual function signature itself, and we can't fix that at the minute because we can't use the spread operator here?

Reply:
@36degrees Yeah, believe me I tried 😭 

Yeah, it doesn't like it because there literally is no `@param`. We use [`arguments`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments) instead.

Did you see I'd added a comment in place of the function params?
(Thought best to try to explain it another way at least)

```mjs
export function mergeConfigs (/* configObject1, configObject2, ...configObjects */) {
```

Parameter spread `(...args)` can be documented in future though

For backwards compatibility we could add Babel to Rollup. It'd pick up the `@babel/plugin-transform-parameters` plugin based on our [**.browserslistrc**](https://github.com/alphagov/govuk-frontend/blob/main/.browserslistrc) config. Transforming `(...args)` → `arguments`

They show before/after examples here:
https://babeljs.io/docs/en/babel-plugin-transform-parameters

Reply:
Let's live with this for now and we can revisit when we look at introducing Babel more widely 👍🏻

Reply:
Unless we want to use an `/* eslint-disable no-alert, no-console */` so we can add the 'correct' JSDoc without the linter complaining about it?

Reply:
We could, but then the `jsdoc` CLI would complain in this PR https://github.com/alphagov/govuk-frontend/pull/2920

Just end up kicking the problem from tool to tool unfortunately